skyportal/handlers/api/group.py

```diff
@@ -349,11 +351,18 @@ def post(self, group_id, *ignored_args):
             return self.error("Cannot add users to single-user groups.")
         user = User.query.filter(User.username == username.lower()).first()
         if user is None:
-            user_id = add_user_and_setup_groups(
-                username=username,
-                roles=["Full user"],
-                group_ids_and_admin=[[group_id, admin]],
+            groups = Group.query.filter(Group.id == group_id).all()
```

Comment:
I am confused. Why is a `User` invitation being generated in `Group.get`?

Reply:
Instead of e.g., an `InvitationHandler`

Reply:
@dannygoldstein this is `GroupUser.post`

Reply:
Sorry I have no idea how I got that mixed up